layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -87,20 +89,27 @@ python do_create_opentrons_manifest() {
 ROOTFS_PREPROCESS_COMMAND += "do_create_opentrons_manifest; "
 
 # add the rootfs version to the welcome banner
-fakeroot do_add_rootfs_version() {
+do_add_rootfs_version() {
     printf "${DISTRO_NAME} ${DISTRO_VERSION} (${DISTRO_CODENAME}) \\\n \\\l\n" > ${IMAGE_ROOTFS}/etc/issue
     printf "${DISTRO_NAME} ${DISTRO_VERSION} (${DISTRO_CODENAME}) %%h\n" > ${IMAGE_ROOTFS}/etc/issue.net
     printf "${IMAGE_NAME}\n\n" >> ${IMAGE_ROOTFS}/etc/issue
     printf "${IMAGE_NAME}\n\n" >> ${IMAGE_ROOTFS}/etc/issue.net
 
+    # add datetime as version, example 202210111213
+    date +"%Y%m%d%H%M" > ${IMAGE_ROOTFS}/etc/version
```

Comment:
ditto here re: hermeticity, let's not put dates in the filesystem if we can help it